src/docs/development/add-to-app/android/add-splash-screen.md

```diff
@@ -9,28 +9,28 @@ development/add-to-app/android/add-splash-screen/splash-screens_header.png
 class="mw-100" alt="Add Splash Screen Header" %}
 
 The beginning of a Flutter experience requires a brief wait while Dart
-initializes. Additionally, a full-Flutter app requires a standard Android app
-initialization time, too. Flutter supports the display of a "launch screen"
-while your Android app initializes, and also supports the display of a "splash
-screen" while your Flutter experience initializes. This guide teaches you how to
+initializes. Additionally, a full Flutter app requires standard Android app
```

Comment:
'full-Flutter' is a term we'd like to coin as the specific alternative to add-to-app